Formula/sonic.rb

```diff
@@ -58,7 +61,7 @@ def plist
     port = free_port
 
     cp etc/"sonic.cfg", testpath/"config.cfg"
-    inreplace "config.cfg", ":1491", ":#{port}"
+    inreplace "config.cfg", "[::1]:1491", "0.0.0.0:#{port}"
```

Comment:
Might be better to try to fix CI?
Part of issues may be related to how we set up docker container.
Not sure if GitHub Ubuntu image has any limitations.

Reply:
@dawidd6 whenever you get the chance, can you look into behavior of binding to ports using IPv6 in current Linux CI?

This is one example where the default IPv6 loopback doesn't work.

Another example is test failure bottling `black`: https://github.com/Homebrew/homebrew-core/runs/3011840997?check_suite_focus=true#step:9:104
```
OSError: [Errno 99] error while attempting to bind on address ('::1', 41913, 0, 0): cannot assign requested address
```

And there are a few other Formulae with similar failures.

Reply:
Never encountered such an issue. Dunno what I could do here. I'm not that good with networking, especially IPv6.

Reply:
If Ubuntu VM supports IPv6, it could be Docker container that isn't set up for IPv6
e.g. https://docs.docker.com/config/daemon/ipv6/

The alternative would be changing failing tests to IPv4 as the CI nodes should support either dualstack IPv4/IPv6 or IPv4-only.

I may need to experiment on fork to check.

Reply:
> The alternative would be changing failing tests to IPv4 as the CI nodes should support either dualstack IPv4/IPv6 or IPv4-only.

I think this is probably the path of least resistance for now.  I'm pretty sure we have other formulae which were not bottled successfully on Linux because of this same issue.  It would be great to eventually fix IPv6 in Docker, but unless we think we're missing something by skipping IPv6 tests on Linux, it would probably be easier to just use IPv4 until we've done that.

Reply:
Sure, that is fine with me.